tests/sql/close.yaml

```diff
@@ -0,0 +1,53 @@
+$schema: ../../json_schemas/test_story.schema.yaml
+
+description: Test to explicitly clear the cursor context
```

Comment:
Add a period at the end.

Reply:
Time for another lint!

Reply:
We have https://github.com/opensearch-project/opensearch-api-specification/issues/403, I'll add a note on tests.